code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -663,8 +663,8 @@
 
 /**
   * # Disk Compartmentalizer
-  *
-  * Disk variant of the [Smart Fridge][/obj/machinery/smartfridge].
+  * # Identification Compartmentalizer
+  * Variants of the [Smart Fridge][/obj/machinery/smartfridge].
   */
 /obj/machinery/smartfridge/disks
```

Comment:
This is an autodoc comment and is specific to that type. Revert this change.

Reply:
Didn't know it was an autodoc comment, it has been reverted.